addons/@ocap/addons/ocap/functions/fn_handleMarkers.sqf

```diff
@@ -32,87 +32,71 @@ ocap_markers_handle = ["ocap_handleMarker", {
 	switch (_eventType) do {
 
 		case "CREATED":{
-			LOG(ARR2("MARKER:CREATE: Processing marker data -- ", _this));
+			if (ocap_isDebug) then {LOG(ARR2("MARKER:CREATE: Processing marker data -- ", _this))};
```

Comment:
Can you turn this into a DEBUG(ARR2 .... instead of LOG?
So we can always call it